regulations/views/preamble.py

```diff
@@ -172,6 +172,41 @@ def make_preamble_toc(nodes, depth=1, max_depth=3):
     ]
 
 
+def section_navigation(full_id, toc, depth=1):
+    toc_flat = [sect for sect in toc if sect.depth == depth]
```

Comment:
Sorry, I see what you're doing now. We're only linking to the top-level sections, so the logic for `next` and `previous` doesn't seem right.